addons/website_payment/static/src/js/link.js

```diff
@@ -0,0 +1,19 @@
+odoo.define("website_payment.Link", function (require) {
+    "use strict";
+
+const Link = require("wysiwyg.widgets.Link");
```

Comment:
Is it possible to use ```import...export``` instead of ```define...require``` for new files?

Reply:
No, ig it is not possible, also include method is different.
and i find no file in 15.0 where anything except define and require is used.

Reply:
I tried following and it works.

```/** @odoo-module **/

import Link from "wysiwyg.widgets.Link";```